.github/workflows/rust.yml

```diff
@@ -64,6 +64,7 @@ jobs:
     - run: cargo clippy --all-targets --target ${{ matrix.target }}
     - run: make build-test-wasms
     - run: cargo install --path ./cmd/soroban-cli --debug
+    - run: cargo test
```

Comment:
The tests for the cli are run a few lines down, in the for loop, because that includes the cli. So I don't think we need this. Please let me know if I'm missing something.